connexion/middleware/exceptions.py

```diff
@@ -65,5 +65,5 @@ def common_error_handler(_request: StarletteRequest, exc: Exception) -> Response
 
     async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
         # Needs to be set so starlette router throws exceptions instead of returning error responses
-        scope["app"] = self
+        scope["app"] = "connexion"
```

Comment:
Including a string here is more lightweight and easier to handle than a module. I believe I changed this when I ran into a pickle error, although I forgot where it originated.